mozfun/event_analysis/extract_event_counts/metadata.yaml

```
@@ -0,0 +1,6 @@
+description: >
+  Extract the events and their counts from an events string.
+
+  This function explicitly ignores event properties, and
+  retrieves just the counts of the top-level events.
+friendly_name: Event_analysis Extract Event Counts
```

```suggestion
friendly_name: Extract Event Counts
```

I don't think repeating the dataset name is necessary in the friendly name.

---

I agree, we should change this default in the CLI.